build/cli/src/arg/wasm.rs

```diff
@@ -57,6 +57,28 @@ impl From<ProfilingLevel> for enso_build::project::wasm::ProfilingLevel {
     }
 }
 
+// Follows hierarchy defined in  lib/rust/logging/src/lib.rs
+#[derive(ArgEnum, Clone, Copy, Debug, PartialEq, Eq)]
+pub enum LogLevel {
```

Comment:
It is good as it is, but the `project/wasm.rs` equivalent can derive `ArgEnum` as well and make this superfluous.
This is a tradeoff between separation (we keep CLI arguments representation separate from the environment variables for wasm) and duplication.